src/schema/v2/previewSavedSearch.ts

```diff
@@ -93,6 +93,14 @@ const PreviewSavedSearchType = new GraphQLObjectType<any, ResolverContext>({
         "URL for a user to view the artwork grid with applied filters matching saved search criteria attributes",
       resolve: resolveHref,
     },
+    suggestedFilters: {
+      description:
+        "Suggested filters for the user to use based on their search criteria",
+      type: new GraphQLList(new GraphQLNonNull(SearchCriteriaLabel)),
```

Comment:
note: I am not sure here if we should allow nulls - I will get to this later when we implement the logic and have a better understanding of what we are going to be returning.

```suggestion
      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(SearchCriteriaLabel))),
```

Reply:
I think we should always be able to return an empty array or non-null suggestions here.

Reply:
Cool - I will do it now then